nextflow_schema.json

```diff
@@ -509,6 +509,21 @@
                     "fa_icon": "fas fa-folder-open",
                     "description": "Specify path to krona taxonomy directories (required for MALT krona plots)",
                     "help_text": "Specify a path to a Krona taxonomy database directory (i.e. a directory containing a krona generated `.tab` file).\n\nThis is only required for generating Krona plots of MALT output.\n\nNote this taxonomy database must be downloaded and generated with the `updateTaxonomy.sh` script from the krona-tools package."
+                },
+                "taxpasta_add_taxonomy": {
```

Comment:
Missing an Icon! And make sure to update the name if you make the changes updated above

Reply:
Do you mean the `fa_icon`?

Reply:
Yes but you'll need to select it via the schema builder